CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ endif()
 
 include(${HPCC_SOURCE_DIR}/cmake_modules/optionDefaults.cmake)
 
-if ( REMBED )
+if ( REMBED AND (NOT INCLUDE_PLUGINS) )
   # various components that are not needed if only making the R plugin
   SET(USE_LIBARCHIVE 0)
   SET(USE_APR 0)
```

Comment:
I checked HPCC 6.2.14-1 build on CentOS 7. htpasswd was built: http://10.240.32.243/view/HPCC-6.2.14-1/job/CE-Candidate-6.2.14-1/CE=centos-7.0-x86_64/2/console.

I will check the packing and installation.